pyright.sublime-syntax

```diff
@@ -0,0 +1,28 @@
+%YAML 1.2
+---
+name: Python (for LSP)
+scope: source.python.lsp
+hidden: true
+
+extends: Packages/Python/Python.sublime-syntax
+
+contexts:
+  main:
+    - meta_include_prototype: false
+    - match: ''
+      push: [statements, lsp-type]
+
+  lsp-type:
+    - meta_include_prototype: false
+    - match: \(
+      scope: punctuation.section.parens.begin.python
+      set: lsp-type-body
+    - match: (?=\S)
+      pop: true
+
+  lsp-type-body:
+    - meta_scope: meta.parens.python
+    - meta_content_scope: storage.type.python
+    - match: \)
+      scope: punctuation.section.parens.end.python
+      pop: true
```

Comment:
Highlighting can still be broken.

<img width="970" alt="Screenshot 2021-12-05 at 19 10 56" src="https://user-images.githubusercontent.com/153197/144758232-f5a318a0-d026-4323-9e30-b2f56a0773e8.png">

Seems like the content in the popup can either be:

(kind) variable_name: type

or 

(kind) type

Possibly more cases of course.

But since this syntax will be used for all cases, even actual python code, we should ensure that we don't break that.

Reply:
Would it be okay to adjust the syntax in another PR?

Reply:
> Would it be okay to adjust the syntax in another PR?

I am fine with this. :D

Reply:
> Highlighting can still be broken.
> 
> <img alt="Screenshot 2021-12-05 at 19 10 56" width="970" src="https://user-images.githubusercontent.com/153197/144758232-f5a318a0-d026-4323-9e30-b2f56a0773e8.png">

This has been fixed in the current master.